modules/phase_field/src/userobjects/DiscreteNucleationMap.C

```diff
@@ -73,30 +71,37 @@ DiscreteNucleationMap::execute()
       Real r, rmin = std::numeric_limits<Real>::max();
 
       // find the distance to the closest nucleus
+      Real local_radius = 0.0;
       for (unsigned i = 0; i < _nucleus_list.size(); ++i)
       {
         // use a non-periodic or periodic distance
         r = _periodic < 0
-                ? (_q_point[qp] - _nucleus_list[i].second).norm()
-                : _mesh.minPeriodicDistance(_periodic, _q_point[qp], _nucleus_list[i].second);
-        if (r < rmin)
-          rmin = r;
+                ? (_q_point[qp] - _nucleus_list[i].center).norm()
+                : _mesh.minPeriodicDistance(_periodic, _q_point[qp], _nucleus_list[i].center);
+        // if (r < rmin)
+        //{
```

Comment:
please remove commented out code

Reply:
Yep.  I got rid of most of them, missed these :( Thanks!